packages/ui-library/src/components/radio/index.stories.ts

```diff
@@ -390,7 +388,7 @@ export const FormCaptionGroup = () => {
         ...args,
         hasHint: true,
         label: 'Hint message',
-        hintIcon: 'blr360',
+        hintMessageIcon: 'blr360',
```

Comment:
Icon should be blrInfo (probably will also be changed within #974 )

Reply:
done